python/src/wrfcloud/runtime/resources/geojson_products.yaml

```
@@ -179,3 +179,32 @@ products:
       - 300
       - 250
       - 100
+
+  - netcdf:
+      variable: wind_speed_10
+    range:
+      min: 0
+      max: 60
```

```suggestion
      max: 120
```
I think we forgot to change this range when we converted m/s --> kts. Speeds go off the top of the color range in the 3D wind speed examples.